blog-cse/2024-11-08-application-update.md

```diff
@@ -0,0 +1,13 @@
+---
+title: November 8, 2024 - Application Update
+keywords:
+  - cloud siem
+image: https://help.sumologic.com/img/sumo-square.png
+hide_table_of_contents: true  
+---
+
+### Cloud SIEM network sensor end-of-life
+
+The Sumo Logic Product Team is discontinuing our on-premise network sensor feature for Sumo Logic Cloud SIEM. The feature will no longer receive updates as of 8 November 2024, and support ends as of 31 January 2025. We fully support a customer or partner managed [Zeek network sensor](/docs/cse/sensors/ingest-zeek-logs/) as a data source for our Cloud SIEM product that will provide equivalent monitoring of your network.
```

Comment:
should we switch to U.S.-style date format?

Reply:
Yes, @kimsauce. Good idea. I switched it.